pkg/sbom/implementation.go

```diff
@@ -95,14 +96,14 @@ func checkEnvironment(spec *Spec) (bool, error) {
 	return true, nil
 }
 
-// generateAPKPackage generates the sbom package representing the apk
-func generateAPKPackage(spec *Spec) (pkg, error) {
+// generateAPKPackage generates the sbom package representing the apk.
+func generateAPKPackage(spec *Spec) (*pkg, error) {
```

Comment:
nit: If I saw this in other code I'd assume it creates the actual APKPackage, again, not your doing, but I'd consider renaming this to generateAPKSBOM or something like that.

Reply:
Agreed 1000%

Reply:
9f3a18f